### PR TITLE
Expose development server network interface

### LINKED ISSUE #170: Idea: Expose development server to network interface
Is it possible to expose the development server to the network interface? This would make it very easy for someone to access his preact app on his phone without having to fiddle around with simulators.

This already works in create-react-app:

> $ yarn start
> Compiled successfully!
> 
> You can now view test in the browser.
> 
>   Local:            http://localhost:3000/
>   On Your Network:  http://192.168.2.104:3000/
> 
> Note that the development build is not optimized.
> To create a production build, use yarn build.

If such a feature already exists, how do I enable it?

### PR DESCRIPTION
**Pull request by reznord**
Exposes network address for development build (fix #170)

This is how it looks now:
image 2017-07-02 at 2 13 33 pm

### CODE REVIEW

File: src/lib/webpack/run-webpack.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import path from 'path';
 import fs from 'fs.promised';
+import ip from 'ip';
```
Comment (thangngoc89) [inline src/lib/webpack/run-webpack.js]: You forgot to install this module

**Comment by reznord** [comment]
@thangngoc89 that package is already available in the node_modules 

here you go:
image 2017-07-02 at 2 54 29 pm

**Comment by thangngoc89** [comment]
@reznord Don't rely on that, we don't know if webpack-dev-server gonna remove it in the future. If we use it directly, import it.

**Comment by reznord** [comment]
okay, will add it to the dependencies

**Pull request merged**